native.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./lib/native');
```

Comment:
This seems to be breaking lint checks on Travis, add the this as the first line above the require statement
`/* eslint-disable import/no-unresolved */`